docs-src/src/pages/menuItems.js

```diff
@@ -0,0 +1,360 @@
+//Items to be displayed in the Navigation 
+
+const menuItems = [
```

Comment:
You may want to consider refactoring this in a more functional manner, using `getMenuItems` and having it return this array, i.e. 
```
const getMenuItems = () => {
    return [
      {
        category: 'Design Tokens'
        ...
```

This way, if we ever change where we are accessing the data (ie a database), we can simply change how this function accesses it. It also makes it easier to build in variables later on if you want to drill down what it is you want to be returned.